src/Test/L0/HostContextL0.cs

```diff
@@ -85,6 +85,8 @@ public void DefaultSecretMaskers()
                 _hc.SecretMasker.AddValue("Pass word 123!");
                 _hc.SecretMasker.AddValue("Pass<word>123!");
                 _hc.SecretMasker.AddValue("Pass'word'123!");
+                _hc.SecretMasker.AddValue("3ch");
+                _hc.SecretMasker.AddValue("{");
```

Comment:
add tests for both `{` and `}` 
make sure `{` won't mask `e` and `}` won't mask `f`